### ISSUE TITLE
Scan multiple pages at once by pointing to sitemap

### ISSUE THREAD

**Issue by molant** [sonarwhal/sonar-tools#6]
When talking to the a11y tools team they mentioned the requirement of being able to scan multiple pages, such as the entire site for issues.
Sites should ideally include a sitemap.xml (or other format, e.g. see https://support.google.com/webmasters/answer/183668?hl=en) for SEO, so it would be great if a user could just point to that file to be able to scan the entire site, rather than having to manually create a list of URLs.
We could even auto-detect the sitemaps file if it is mentioned in the robots.txt file and offer to scan that rather than just homepage.

- - - - - - - - - - -

Original request by @dstorey in https://github.com/sonarwhal/sonar/issues/383

**Comment by alrra** [comment]
@molant Can you explain a bit more why this issue was moved tot this repository? Thanks!

**Comment by molant** [comment]
This request is similar to what is being done with crawler: We scan multiple urls based on an input file.

I think that all of those small scripts that are interesting but not essential should live in a separate repository.

**Comment by alrra** [comment]
> This request is similar to what is being done with crawler: We scan multiple urls based on an input file.

My thinking was that in addition to using the `sitemap.xml` we would just follow same origin links as we encounter them in the initially provided page (and that kinda requires things to be inside sonar).

**Comment by molant** [comment]
> we would just follow same origin links as we encounter them in the initially provided page

I don't think we should crawl like that. Process could take forever and I'm not sure it adds a lot of value to have it inside sonar. If a user wants to know what urls the website has, there're already tools for that and IMO  it would be perfect for a script in this repo.

**Comment by alrra** [comment]
> If a user wants to know what urls the website has, there're already tools for that and IMO it would be perfect for a script in this repo.

:+1:
